src/libprojectM/PCM.hpp

```diff
@@ -31,6 +31,11 @@
 
 #include "dlldefs.h"
 
+
+// 1024 is more computationally intensive, but maybe better at detecting lower bass
+#define FFT_LENGTH 512
```

Comment:
would be cool if this was variable, based on the current audio input sample rate some day